modules/manage/pages/cluster-maintenance/compaction-tuning.adoc

```diff
@@ -0,0 +1,48 @@
+
+*  When using tiered storage, compaction functions at the local level before segments are uploaded to storage. Once a segment is uploaded to storage it is not retrieved for further compaction operations. A key may therefore appear in multiple segments between tiered and local storage. 
+* Compaction on Redpanda is bounded by raft leadership terms. The same key may appear across multiple raft terms, as shown in the preceding diagram. 
+
+While compaction reduces storage needs, Redpanda's compaction (just like Kafka's) does not guarantee perfect de-duplication of a topic. Duplicates of a message, or additional messages, may still exist within a topic ordered by time. Compaction is not a complete topic operation; rather, it represents a best effort mechanism to reduce storage needs for topics created within a certain time window.
+
+== Configuring Cleanup Policy
+
+You typically configure compaction policy at the cluster level, but you can also specify an individual topic policy to override the cluster's setting. The cluster-level xref:reference:cluster-properties.adoc#_log_cleanup_policy[`log_cleanup_policy`] and the topic-level xref:reference:topic-properties.adoc#cleanuppolicy[`cleanup.policy`] support the following three options:
+
+* `delete`: Messages are deleted from the topic once the specified retention period (time and/or size allocations) is exceeded. This is the default mechanism and is analogous to disabling compaction.
+* `compact`: This triggers only cleanup of messages with multiple versions. A message that represents the only version for a given key is not deleted.
+* `compact,delete`: This combines both policies, deleting messages exceeding the retention period while compacting multiple versions of messages.
+
+== Compaction Policy Settings
+
+The various cleanup policy settings rely on proper tuning of a cluster's compaction and retention policy options. The applicable settings include the following:
+
+* xref:reference:cluster-properties.adoc#_log_compaction_interval_ms[`log_compaction_interval`]: Defines the compaction frequency in milliseconds. (default: 10,000ms)
+
+* xref:reference:tunable-properties.adoc#_compaction_ctrl_backlog_size[`compaction_ctrl_backlog_size`]: Defines the size for the compaction backlog of the backlog controller. (default: 10% of disk capacity)
+
+* xref:reference:tunable-properties.adoc#_compaction_ctrl_min_shares[`compaction_ctrl_min_shares`]: Defines the minimum number of I/O and CPU shares the compaction process can use. (default: 10)
+
+* xref:reference:tunable-properties.adoc#_compaction_ctrl_max_shares[`compaction_ctrl_max_shares`]: Defines the maximum number of I/O and CPU shares the compaction process can use. (default: 1,000)
+
+* xref:reference:tunable-properties.adoc#_storage_compaction_index_memory[`storage_compaction_index_memory`]: Defines the size for the compaction backlog of the backlog controller. (default: 10% of disk capacity)
```

Comment:
The description on this one seems off: https://github.com/redpanda-data/redpanda/blob/8837ad176a450dba57a4fbef6e47e15e1a56a88c/src/v/config/configuration.cc#L1044